templates/taxi/driver_detail.html

```diff
@@ -23,4 +26,5 @@ <h4>Cars</h4>
       <p>No cars!</p>
     {% endfor %}
   </div>
+  <a href="{% url 'taxi:driver-delete' pk=driver.id %}" type="button" class="btn btn-danger">Delete</a>
```

Comment:
Add Cancel button apart from Delete one. The Cancel button will lead to the previous page the user was on.